components/forms/FormDB.js

```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-export default function Form({ onAddData }) {
+export default function Form({ onSubmit }) {
```

Comment:
it should be Form({onSubmit, days}) right? since u call <FormDB onSubmit={handleSubmit} days={days} /> in ur addData.js

Reply:
I just noticed, that I'M not using that - at all. Its was for future usage, but I took it out for now